mono/eglib/test/test-eglib.exp

```diff
@@ -0,0 +1,3 @@
+* This file is required so that the symbol is visibile and not GCed on AIX.
+dummy_test_export
+
```

Comment:
Yeah a .def file! :)

Reply:
I went with what the AIX linker uses for convention; but it is kinda like that, heh.